hack/tests/ci-operator-prepare.sh

```diff
@@ -15,10 +15,20 @@ set_build_images() {
 }
 
 setup_secrets() {
-    ln -sf $SECRET_DIR secrets
+    if [[ -e /usr/local/e2e-secrets/azure ]]; then
+        ln -sf /usr/local/e2e-secrets/azure secrets
+    fi
```

Comment:
this will break local development because you aren't creating a link from the github.com/openshift/openshift-azure to the versioned git repo that has been cloned into /tmp